pythonforandroid/recipes/Pillow/__init__.py

```diff
@@ -45,6 +45,7 @@ def get_recipe_env(self, arch=None, with_flags_in_cc=True):
         cflags += ' -I{}'.format(free_inc_dir)
         cflags += ' -I{}'.format(jpeg_inc_dir)
         cflags += ' -I{}'.format(ndk_include_dir)
+        cflags += ' -fPIC'
```

Comment:
I see that you add this "position independent code" flag (`-fPIC`) for some recipes. We already set this flag for some archs, so it would be redundant when building for that archs...:thinking:...I'm wondering if we should be brave and set it at [arch level](https://github.com/kivy/python-for-android/blob/develop/pythonforandroid/archs.py#L255-L303)...what do you think guys?

Reply:
BTW, I tested it locally setting the `-fPIC` flag at arch level for `arm64-v8a` and it seems to work fine at build and runtime (using `on_device_unit_app` with matplotlib recipe...which triggers the build of some libraries and recipes)

Reply:
I think, if using a recent NDK, that should be safe. So I would vote for adding them at the Arch level. Should I do it?

Reply:
I tested it locallu for `armv7` `armv8-64 ` and `x86` and it worked.

Reply:
Well, we will have less code and that is good, so unless @AndreMiras says the contrary, I also would be brave and set it at arch level...

Reply:
I don't have anything against that. And we will know soon enough if it creates runtime issues :+1: